includes/polling/os/generex-ups.inc.php

```diff
@@ -0,0 +1,6 @@
+<?php
+
+$hardware_array = snmp_get_multi($device, 'upsIdentManufacturer.0 upsIdentModel.0', '-OQUs', 'UPS-MIB');
+d_echo($hardware_array);
+$hardware = trim($hardware_array[0]['upsIdentManufacturer'], '"') . ' - ' . trim($hardware_array[0]['upsIdentModel'], '"');
+$version  = snmp_get($device, 'upsIdentAgentSoftwareVersion.0', '-OQv', 'UPS-MIB');
```

Comment:
Move this to the multi get above, or are specific parameters used in the get required?